arxiv_latex_cleaner/arxiv_latex_cleaner.py

```diff
@@ -212,7 +212,7 @@ def _remove_iffalse_block(text):
 
 def _remove_comments_inline(text):
   """Removes the comments from the string 'text' and ignores % inside \\url{}."""
-  if 'auto-ignore' in text:
+  if '%auto-ignore' in text.split():
```

Comment:
I would generalize this a bit more to accept spaces in between the `%` and `auto-ignore`. It can be something like `if text.replace(' ', '').startswith('%auto-ignore')`.

Reply:
@jponttuset I didn't allow spaces between '%' and 'auto-ignore' because the arXiv documentation doesn't clarify if it's allowed. And the examples I saw all followed this format.

Reply:
I'd still be a bit more lenient here and allow spaces. One problem I now see, though, is that this keeps the whole line, and you might have a super-secret message right after auto-ignore :)
So maybe I'd do:
```
if text.replace(' ', '').startswith('%auto-ignore'):
  return '%auto-ignore'
```

Reply:
We should also consider the possibility of %auto-ignore being in the middle of a line. The new commit I just pushed takes this into account as well.